### ISSUE TITLE
Two-Headed Dragons/“Hydraborn”

### ISSUE THREAD

**Issue by kyanightfury** [AlexModGuy/Ice-and-Fire-Suggestions#1279]
TL;DR: A rare variant of Ice/Fire/Lightning dragons that possesses a second head—a la the rare gila-monster-esque variant of the Royal Red Dragon in the Wyrmroost mod, except without a differential texture for the entity; it would simply be a head and neck duplication. (Although I know ABSOLUTELY NOTHING about how Minecraft mods are made, so it might be just as complicated, if not more so than making a new texture, especially given the new/modified animations potentially involved; I have no idea.) Also, if someone has already posted a two-headed dragons suggestion, I’m sorry for stepping on your toes; that was not at all my intention. I also apologize for any potential spam if a suggestion in this vein has already been closed/rejected. I don’t have time to check on either of those things, so I can only apologize for any confusion/annoyance there.

Further description/possible lore to accompany the idea:

Okay, so this might sound quite ridiculous, but I got really attached to the idea of a two-headed dragon when my sister sent me a cool video of a little two-headed tortoise. (Specifically, I wanted the ability to hatch an electric blue lightning dragon with a fancy name for each dragon head—Kyanyxus and Kairulanthor, named after some lightning dragons from my recently abandoned survival longplay who sadly despawned.) In nature, this occurs when a developing embryo begins to split into identical twins but stops midway through the process, leaving the twins joined. 

After quickly learning that despite my skills in making mods work in Minecraft, I would be COMPLETELY LOST trying to implement such a feature in an add-on mod, I did a bit more research into the idea of a two-headed dragon for this post. The concept struck me as particularly interesting because polycephaly/bicephaly occurs more often in reptiles (especially snakes and tortoises/turtles/terrapins) than any other animal, and dragons are most commonly understood as reptilian (or in older mythology, often serpentine) or at least archosaurian/dinosaur-adjacent. 

Typically, when I think of a two-headed dragon, a Zippleback from HTTYD or something similar comes to mind: a species of dragon that is *always* born with two heads. However, in actual nature it’s more of a rarity, and I just think it would be cool if that sort of rarity were incorporated into the existing mod dragons. 

Mechanically, it could work like this: in the mod, there would a *very small* chance that an egg would hatch a dragon with two heads—otherwise known as a Hydraborn dragon. This chance would be slightly raised when breeding a one-headed dragon with a Hydraborn, and guaranteed when crossing two Hydraborn. Alternatively, a player could come across an ancient Hydraborn (more on this later), whose egg(s) will ALWAYS be born with two heads. These dragons could have some kind of double damage and/or health potential—or at least a wider damage hit box—and the second head/neck in a tamed Hydraborn could be given its own name, helmet, and neck armor. This could be a good alternative to adding another kind of dragon to the mod (which I know will most likely never happen, but it’s still something the community buzzes about occasionally). A Hydraborn dragon would also be much more difficult for a player to kill, with (maybe) increased health and damage, so such a feature could serve as a rare bonus challenge for players who enjoy intense bosses.

This concept could also have somewhat interesting lore attached to it (this is where the name “Hydraborn” comes in). I was screwing around with this mod in creative one day as I sometimes do and spawned in a few hydras. (I was curious what they looked like since I’ve yet to encounter one in survival.) I then I spawned in some Lightning dragons and watched them fight. I was surprised to see how much damage the hydra did, even quickly dispatching a few of the younger dragons (although it does make sense given my usual survival strategy of stocking up on hydra fangs via Scribe villager trades when I want to go dragon hunting).

This gave me the sense that dragons might have a natural fear of hydras: the only non-draconic creatures that really pose a threat to them (apart from gorgons, I suppose). This could make Hydraborn dragons even rarer in the wild. Adult dragons would reject their Hydraborn offspring, fearing their multi-headed nature made them as dangerous as a hydra (hence the name Hydraborn). Stage 3 Hydraborn would be killed off by groups of other stage 3 dragons. Thus, the only Hydraborn dragons found in the wild would be stage 4 or 5; underground where no other dragon could harm them, having lived their entire lives alone.

Anyway, that’s about the end of this post. This is my first GitHub suggestion, so I’m not sure if these things can be updated, but I might try to add a drawing of the dragon I had envisioned as proof of concept later on. Thank you for reading this far!! And mod devs, if you see this, thank you so much for making this mod. I’ve been following it since it was in its first round of development; I grew up with it and it’s brought a lot of much-needed joy to my life, especially this year.
